NSS-Day-Cohort-53/tabloid-full-stack-always-puffy-impalas
# Hp register new user

#### Changes Made
1. Modified file `Register.js` to include `register` functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout HP-registerNewUser
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.

3. Test app functionality.
    > To Test  the register function, logout and click register under the input fields on the login page. This should redirect you to a register page where you enter all the personal information a new user needs. Upon filling in these fields, the user should be redirected to the home page and the database should now have the user you just created.

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

Closes #19
